docs/DeveloperGuide.adoc

```diff
@@ -337,17 +339,115 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. DukeCooks shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+[discrete]
+=== Use case: Suggest Food
+
+*MSS*
+
+1.  User requests for a food suggestion
+2.  DukeCooks gives a food recommendation
+3.  User is satisfied with the recommendation
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. User is unsatisfied with the food recommendations.
++
+[none]
+** 3a1. User asks for another suggestion.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Add Calorie Intake
+
```

Comment:
Maybe consider clarifying the information provided in Nutrition (Breakdown)? Its slightly unclear what the list of Nutrition means